### DISCUSSION TITLE
save as PNG if transparent

### DISCUSSION THREAD

**Discussion by Halfjog** [chaiNNer-org/chaiNNer#2810] [Q&A]
hello i want to convert a bunch of tga files in a folder: 
i want to save it as jpg if it doesnt have an alpha channel, and in png if it does
can i do this with chainer? or what other software do you recommend

**Comment by joeyballentine** [comment]
There is currently no conditional branching support in chaiNNer so that is not something you can accomplish unfortunately. I'm not sure what other software would be able to do that though, besides just writing it as a python script

**Comment by Halfjog** [comment]
ok well if anyone needs to do this i found a program called ImBatch
it can filter and sort images
